packages/components/collapse/src/Collapse.tsx

```diff
@@ -61,6 +62,20 @@ export const Collapse = ({
     };
 
     if (current) {
+      // Don't animate on first render
+      if (isMounted.current) {
```

Comment:
Shouldn't this be the other way around?

Reply:
No, there is a transition already on the `.styles` file, that we override with the 0s transition, that we remove here.